src/ng/directive/ngModelOptions.js

```diff
@@ -423,8 +417,72 @@ defaultModelOptions = new ModelOptions({
  * Default events, extra triggers, and catch-all debounce values}.
  *
  *
+ * ## Specifying timezones
+ *
+ * You can specify the timezone that date/time input directives expect by providing its name in the
+ * `timezone` property.
+ *
+ *
+ * ## Formatting the value of `time` and `datetime-local`
+ *
+ * With the options `timeFormat` and `timeStripEmptySeconds` it is possible to adjust the value
+ * that is displayed in the browser control. Note that browsers may apply their own formatting
+ * in the user interface.
+ *
+   <example name="ngModelOptions-time-format" module="timeExample">
+     <file name="index.html">
+       <time-example></time-example>
+     </file>
+     <file name="script.js">
+        angular.module('timeExample', [])
+          .component('timeExample', {
+            templateUrl: 'timeExample.html',
+            controller: function() {
+              this.time = {
```

Comment:
Does this need to be an object?

Reply:
No, but it's good practice ;)

Reply:
<sub>_Disagrees, but not strongly enough to be bothered 😛_</sub>

Reply:
(it is only good practice if you are working directly with the scope - in this case you already have an object on the scope `$ctrl` so it doesn't matter)